blocked-edges/4.13.25-NewMachineSetForNodesNotWorking.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.13.25
+from: 4[.]12[.].*
+url: https://issues.redhat.com/browse/MCO-958
+name: NewMachieSet-ForWorkerNode-NotWorking
+message: |-
+  Adding a new worker node will fail for clusters running on ARO.
```

Comment:
I think this may affect nodes on reboot but we don't know it yet, anyway this is fine for now and we can amend it later.